src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -624,6 +630,21 @@ public void onLivingUpdate()
             pickupItems();
             cleanupChatMessages();
             updateColonyServer();
+            if(worldObj.isDaytime())
+            {
+                SoundUtils.playRandomSound(worldObj, this);
+            }
+            else if(worldObj.isRaining() && 1 <=rand.nextInt(RANT_ABOUT_WEATHER_CHANCE))
```

Comment:
I'd suggest we add a todo to change this from fisherman to whatever citizen it actually is.